### PR TITLE
Fix editing collection details and save to server

### PR DESCRIPTION
**Pull request by caneruguz**
Fixes JIRA Ticket OC-27 
https://openscience.atlassian.net/browse/OC-27

Testers will need to change the headers sent for Authorization in local environment

### CODE REVIEW

File: client/app/adapters/application.js [hunk 1]
```diff
@@ -2,5 +2,8 @@ import DS from 'ember-data';
 
 export default DS.JSONAPIAdapter.extend({
     host: 'http://127.0.0.1:8000',
-    namespace: 'api'
+    namespace: 'api',
+    headers :{
+        'Authorization' : 'Basic ASDASDASDASDASD'
```
Comment (samanehsan) [inline client/app/adapters/application.js]: We can do something like this for settings that are not supposed to be committed (define it in environment.js and set it as an environment variable locally): https://github.com/CenterForOpenScience/isp/blob/develop/config/environment.js#L55

Comment (caneruguz) [inline client/app/adapters/application.js]: Nice! Thank you!

File: client/app/components/collection-detail.js [hunk 2]
```diff
@@ -2,20 +2,34 @@ import Ember from 'ember';
...
-            let tags = this.get('modelCache.attributes.tags').split(',');
-            this.set('model', Ember.copy(this.get('modelCache'), true));
-            this.set('model.attributes.tags', tags);
-            this.set('modelCache', null);
+            let model = this.get('model');
+            model.set('title', this.get('modelCache.title'));
+            model.set('description', this.get('modelCache.description'));
+            model.set('tags', this.get('modelCache.tags'));
+            model.save();
+            // let tags = this.get('modelCache.attributes.tags').split(',');
```
Comment (samanehsan) [inline client/app/components/collection-detail.js]: Can you also remove this commented code?

**Comment by samanehsan** [comment]
Looks good to me! :heart:

**Pull request merged**